css/src/components/persona.scss

```diff
@@ -0,0 +1,82 @@
+$persona-font-size-xs: $font-size-9 !default;
+$persona-font-size-sm: $font-size-8 !default;
+$persona-font-size-md: $font-size-7 !default;
+$persona-font-size-lg: $font-size-6 !default;
+$persona-font-size-xl: $font-size-5 !default;
+
+$persona-avatar-background-color: $alternate-background-medium !default;
+$persona-avatar-font-color: $text-invert !default;
+$persona-avatar-border-radius: $border-radius-rounded;
+
+$persona-name-font-size: $font-size-7 !default;
+$persona-name-font-color: $text !default;
+
+$persona-metadata-font-size: $font-size-9 !default;
+$persona-metadata-font-color: $text-subtle !default;
+
+$persona-gap-size: $layout-1 !default;
+
+.persona {
+	display: flex;
+	position: relative;
+	gap: $persona-gap-size;
+
+	.persona-avatar {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		width: 2.35em;
+		height: 2.35em;
+		border-radius: $persona-avatar-border-radius;
+		background-color: $persona-avatar-background-color;
+		color: $persona-avatar-font-color;
+		font-size: $persona-font-size-md;
+		line-height: 1;
+		text-transform: uppercase;
+
+		img,
+		svg {
+			display: block;
+			width: 100%;
+			height: auto;
+			max-height: 100%;
+			border-radius: $persona-avatar-border-radius;
+		}
+	}
+
+	.persona-details {
+		align-self: center;
+		color: $persona-metadata-font-color;
+		font-size: $persona-metadata-font-size;
```

Comment:
Shouldn't font-size be based on em here?

Reply:
I finally got what the confusion was - the font variation I made was affecting the initials only to fit nicely in whatever avatar size is, and the description always had the same size. That's why I didn't use `em` here.

Reply:
I can make description sizes change as well, will look like this:

![image](https://github.com/microsoft/atlas-design/assets/57374379/3731cf18-db73-4eed-9da6-9e95854ee9c2)

Reply:
After thinking about it a little more, for correct relative calculation the difference between sizes should be the same. So we should either proceed with keeping the persona's details part (sorry, not description as I said before) font the same for all sizes or update the `36` -> `40`. It's a small change and used only in one spot.

Reply:
Updated again, since we have the `32` option used everywhere but global nav, I made its sizing perfect (avatar 32px, name 16px, and meta 12px), and the other two variants scale keeping the same proportions though a little different from the initial design: medium - avatar 37.8px, name 18.9px and meta 14px, large - avatar 43.19px, name 21.6 and meta 16px 😅 

![image](https://github.com/microsoft/atlas-design/assets/57374379/a59181d2-fc91-480a-a32b-5efb3dede04f)